core/framework/extractors/robust_extractor_backup_20250913.py

```
@@ -0,0 +1,198 @@
+
+
+def _extract_with_docling(pdf_path: str, use_ocr: bool, extract_tables: bool) -> Dict[str, Any]:
+    """Extract using Docling (runs in subprocess)."""
+    try:
+        extractor = DoclingExtractor(
+            use_ocr=use_ocr,
+            extract_tables=extract_tables,
+            use_fallback=False  # Don't use fallback here, we handle it at higher level
+        )
+        return extractor.extract(pdf_path)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)